prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -136,7 +137,7 @@ class ReportFunction : Logging {
 
             if (authenticationStrategy is OktaAuthentication) {
                 // The report is coming from a sender that is using Okta, so set "oktaSender" to true
-                return authenticationStrategy.checkAccess(request, senderName, true) {
+                return authenticationStrategy.checkAccess(request, senderName, true, actionHistory) {
```

Comment:
Passing actionHistory to the okta checking function enables us to easily expand action history tracking to all places where humans are modifying the system.